src/TConfig.cpp

```diff
@@ -32,6 +33,13 @@ TConfig::TConfig() {
     }
 }
 
+void WriteSendErrors(const std::string& name) {
+    std::ofstream CfgFile { name, std::ios::out | std::ios::app };
+    CfgFile << "# If SendErrors is `true`, the server will send helpful info about crashes and other issues back to the BeamMP developers. This info may include your config, who is on your server at the time of the error, and similar data. You can opt-out of this system by setting this to `false`."
```

Comment:
We need to add an option to hide this text as well. Just for those who are picky :P

Reply:
Yeah, on it right now - that's why its still a draft PR. Anything else I missed that we wanted to put?